src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/FixedActiveThreadsExecutor.kt

```diff
@@ -201,21 +194,43 @@ internal class FixedActiveThreadsExecutor(private val nThreads: Int, runnerHash:
         return null
     }
 
+    private fun updateAdaptiveSpinCount() {
```

Comment:
Please update the branch to the recent `develop` -- we no longer count these thresholds adaptively.